perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -865,7 +868,11 @@ def Prepare(self, vm):
 
     vm.bucket_name = 'pkb%s' % FLAGS.run_uri
 
-    vm.RemoteCommand('%s mb gs://%s' % (vm.gsutil_path, vm.bucket_name))
+    make_bucket_command = '%s mb' % vm.gsutil_path
+    if FLAGS.object_storage_gcs_multiregion:
```

Comment:
Let's output the gcs multi-region location name, and the regional location name (of all 3 providers) as a metadata, for example - 

metadata['gcs_multiregion_location'] = "default", or whatever the user has specified via the flag.

metadata['regional_bucket_location'] = "default", or whatever the user has specified via FLAGS.object_storage_region (for all 3 providers)

This way, when pkb users collect and analyze data from each run, they can categorize based on bucket locations and compare properly.